collections/Tanner/MS_Tanner_346.xml

```diff
@@ -37,57 +40,203 @@
                   </altIdentifier>
                </msIdentifier>
+               <head>Anthology of courtly poetry in English; England, s. xv. med</head>
                <msContents>
+                  <textLang mainLang="en">Middle English</textLang>
+                  <msItem>
+                     <locus>(fol. i r – ii v)</locus>
+                     <note>Modern endleaf, blank.</note>
+                     </msItem>
+                  <msItem><locus>(fol. iii r)</locus>
+                     <note>Modern endleaf, table of contents, headed ‘Some of Chaucer’s Works’, 17th century.</note>
+                     </msItem>
+                  <msItem>
+                     <locus>(fol. iii v – iv v)</locus>
+                     <note>Modern endleaf, blank.</note>
+                  </msItem>
                   <msItem n="1" xml:id="MS_Tanner_346-item1">
-                     <author key="person_100185203">Geoffrey Chaucer</author>
-                     <title key="work_1665">poems</title>
-                     <textLang mainLang="en">English</textLang>
+                     <locus>(fols. 1r-40v)</locus>
+                     <author key="person_100185203"><supplied>Geoffrey Chaucer</supplied></author>
+                     <title key="work_6329"><supplied>The Legend of Good Women</supplied></title>
+                     <incipit>A thousand tymes I hau<supplied reason="damage">e</supplied> <supplied reason="damage">he</supplied>rde men tell <lb/> That the<supplied reason="damage">r</supplied> is ioy in h<supplied reason="damage">euen</supplied> ⁊ peyne in hell</incipit>
+                     <explicit>And til she was caughte and fetyrde in pryson <lb/> This tale is sey for this conclusion</explicit>
+                     <bibl type="repertory"><ref target="https://www.dimev.net/Results.php?imev=100">IMEV 100</ref></bibl>
                   </msItem>
                   <msItem n="2" xml:id="MS_Tanner_346-item2">
-                     <author key="person_9902409">Thomas Hoccleve</author>
+                     <locus>(fols. 41r-48v)</locus>
+                     <author key="person_9902409"><supplied>Thomas Hoccleve</supplied></author>
                      <title key="work_4808">Lespistre de Cupide</title>
-                     <textLang mainLang="en">English</textLang>
+                     <rubric>Incipit littera Cupidinis dei Amoris directa <lb/> subditis suis Amatoribus</rubric>
+                     <incipit>Cupido vn to whos <sic>comaumdement</sic> <lb/> The gentil kinrede of goddis an hy</incipit>
+                     <explicit>The yere of grace ioyfull and iocunde <lb/> A thowsande and foure hundred and seconde</explicit>
+                     <finalRubric>Explicit littera Cupidinis dei amoris <lb/> directa subditis suis amatoribus</finalRubric>
+                     <listBibl><bibl>ed. Frederick James Furnivall, <title>Hoccleve’s Works: The Minor Poems</title>, 3 vols (vol. 1.), EETS e.s. 61 (1892)</bibl>
```

Comment:
I would think it unnecessary to include reference to an edition, since this is already covered through the DIMEV citation.